types/jsdom/base.d.ts

```diff
@@ -39,7 +39,7 @@ declare module "jsdom" {
          *
          * @throws {Error} If the JSDOM was not created with `includeNodeLocations`
          */
-        nodeLocation(node: Node): ElementLocation | null;
+        nodeLocation(node: Node): Token.ElementLocation | null;
```

Comment:
This should probably be `Location | null`  and import Location from parse5, because that is how sourceCodeLocation property seems to be typed in parse5 sources. https://github.com/inikulin/parse5/blob/8e22fe40d5774e2f2d5eb9be230f4f78d60e780d/packages/parse5/lib/tree-adapters/default.ts#L24
and its returning that property in JsDOM v20 implementation  https://github.com/jsdom/jsdom/blob/04f6c13f4a4d387c7fc979b8f62c6f68d8a0c639/lib/api.js#L71


ref: https://github.com/jsdom/jsdom/blob/6d7ccf2517259fe97b082b363134ea9612800747/Changelog.md#2000

Reply:
Seems top level type for "Token" is internal, but it could be used directly from dist folder: 
import { Location } from "parse5/dist/common/token";

Reply:
yes `Token` is currently listed as `@internal`, but it is also re-exported on the entry file https://github.com/inikulin/parse5/blob/842564abf56066dbf4ac07b271bb9baf1d710d97/packages/parse5/lib/index.ts#L16-L17

Reply:
change to `Location` in 76155c1e4738b64f412bd803a09a131e992c07fd

Reply:
To me it sounds weird design from parse5 developers that they have made the type internal, but the type is anyway exposed through public properties

Reply:
changed to directly import from file in cb9ca0596734c7f4ab928b277991896693b6cbfe